### PR TITLE
LPS-163829 Add feature flag to Liferay Experience Cloud onboarding

### PR DESCRIPTION
**Pull request by wandersonvicente**

### CODE REVIEW

**Comment by wandersonvicente** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/index.js [hunk 1]
```diff
@@ -112,7 +113,7 @@ const Pages = () => {
 
 	const StepsLayout = {
 		[ONBOARDING_STEP_TYPES.invites]: {
-			Component: (
+			Component: Liferay.FeatureFlags['LPS-153478'] && (
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/index.js:116]: A Feature Flag eh no InviteTeamMember?

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/index.js (hunk 1 above)
Comment (wandersonvicente) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/index.js]: Alteração feita.

**Comment by wandersonvicente** [comment]
ci:test:sf

**Review by claricebrasil — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward

**Comment by jairmedeiros** [comment]
ci:forward

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**